scripts/zkai_relabeling_benchmark_suite.py

```
@@ -0,0 +1,344 @@
+                f"command: {' '.join(cmd)}\nstdout:\n{completed.stdout}\nstderr:\n{completed.stderr}"
+            )
+        payload = json.loads(completed.stdout)
+    if payload.get("schema") != RUST_ADAPTER_SCHEMA:
+        raise RuntimeError(f"unexpected rust adapter schema: {payload.get('schema')!r}")
+    raw_results = {
+        item["case_id"]: (bool(item["accepted"]), str(item["error"]))
+        for item in payload["results"]
+    }
+    baseline_accepted, baseline_error = raw_results.pop("baseline")
+    if baseline_error:
+        raise RuntimeError(f"baseline fixture failed under Rust production adapter: {baseline_error}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)